content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -617,3 +617,64 @@ Server Side Apply is a beta feature, so it is enabled by default. To turn this
 you need to include the `--feature-gates ServerSideApply=false` flag when
 starting `kube-apiserver`. If you have multiple `kube-apiserver` replicas, all
 should have the same flag setting.
+
+## Resource Versions
+
+Resource versions are strings that identify the server's internal version of an object. Resource versions can used by clients to determine when objects have changed, or to express data consistency requirements when getting, listing and watching resources. Resource versions must be treated as opaque by clients and passed unmodified back to the server. For example, clients must not assume resource versions are numeric, and must not compare two resource versions for greater-than or less-than relationships.
```

Comment:
nit: s/can used/can be used/

Maybe also: ... numeric, and can compare two resource version for equality (i.e. must not compare them for greater-than or less-than relationships).

Reply:
Thanks, I've updated the text to match.